www/js/app.js

```diff
@@ -386,6 +397,36 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         }, 600);
     });
 
+    //focus search bar (#prefix) if Home key is pressed
+    function focusPrefixOnHomeKey(event) {
+        //check if home key is pressed
+        if (event.key === 'Home') {
+            //scroll to top of #search-article section if scroll bar is associated with it
+            $('#search-article').scrollTop(0);
```

Comment:
@Hrishi-Raj Could I ask why you are trying to scroll the search-article element when it is not scrollable?
jQuery swallows an error here. If you convert to native JS:

`document.getElementById('search-article').scrollTo(0, 0);`

you'll see that IE11 throws an error that the element is not scrollable.

Reply:
Maybe you instead want to scroll some element into view? For that there is the native command:

`document.getElementById('someElement').scrollIntoView({behavior: 'smooth'});`